Testing/TestProcessorClasses.hh

```diff
@@ -7,6 +7,7 @@
 
 #include "PrimaryProcessor.hh"
 #include "Processor.hh"
+#include "QuitChain.hh"
 #include "Signal.hh"
 #include "Slot.hh"
 
```

Comment:
Is there any requirement to update this class if any changes are made to the main processor classes? What if I add a new processor, does it get tested or do I have to make a new Test new processor class?

Reply:
The `TestProcessorClasses.hh` file is really just aimed at having some simple Processors that can be used to test specific features of Nymph.  If other test Processors are needed, they could be added here.

In general, for instance when adding Processors to Katydid, they'd go in their own header and source files, and then there'd be a `Test[my processor].cc` file that includes tests dedicated to the functionality of the Processor itself (rather than testing the functionality of the framework).